console/console.go

```diff
@@ -138,7 +150,10 @@ func (c *Console) init(preload []string) error {
 	// Load the supported APIs into the JavaScript runtime environment
 	apis, err := c.client.SupportedModules()
 	if err != nil {
-		return fmt.Errorf("api modules: %v", err)
+		if !isMethodNotFoundError(err) {
+			return fmt.Errorf("api modules: %v", err)
+		}
+		apis = apisAll
```

Comment:
@hyunsooda Do u mean ignore error and  print logs if failed to call `SupportedModule()` ?

Reply:
I'm under the impression that you tried to enable some of the RPC modules (in `apisAll` variable) since there's no control of the interesting RPC module turning on. However, the modules seem not actually turned on by this change. It seems just print log (`use all modules because "rpc_modules" does not supported`)

Reply:
@hyunsooda The modules are activated by `init()` function and `Welcome()` just prints messages with possible modules i.e we can use all modules. But i added welcome messages if failed to call `rpc_modules` like below

```
 modules: admin:1.0 debug:1.0 eth:1.0 klay:1.0 net:1.0 personal:1.0 rpc:1.0 txpool:1.0 web3:1.0
```

Reply:
Your change:
> - use all api modules if "rpc_module" not found in the console
> - add header flags for console

I got the second one, but the first one seems not correct because the client seems not to have the control to enable modules the client want to turn on. Even if you replace the `api` variable to `apisAll`, it is a local change, not the change in the remote node that services.

If I'm missing something, could you explain the output you attached?
```
$ ./ken attach https://node-api.klaytnapi.com/v1/klaytn --header 'x-chain-id:8217' --header 'Authorization:Basic S0...'
Welcome to the Klaytn JavaScript console!

instance: Klaytn/v1.8.3/linux-amd64/go1.18
use all modules because "rpc_modules" does not supported

> klay.blockNumber
94325753

> admin.peers
Error: 400 Bad Request: {"code":1034210,"message":"Unsupported method - admin_peers","requestId":"ac28ef9a-9dd7-9793-83c1-49ec16b6d214"}
    at web3.js:3278:20
....
```
Even though the `admin` module is associated to `apiAll` variable, but not actually turned on.

Reply:
@hyunsooda It is just enable to call "admin" modules in the client side not server side and will receive 400 Bad Request because rpc server does not support "admin" modules.

As u know, Infura and KAS(Klaytn api service) does not support "rpc_modules" rpc. So we cannot use the `console` module because cosnole are required "rpc_moduels" to query available modules.

This PR is just enable to call any modules even though we receive bad request like `> admin.peers
Error: 400 Bad Request: {"code":1034210,"message":"Unsupported method - admin_peers","requestId":"ac28ef9a-9dd7-9793-83c1-49ec16b6d214"}
    at web3.js:3278:20
....`